### DISCUSSION TITLE
Problem with setting network location

### DISCUSSION THREAD

**Discussion by teslasystemsx** [PowerShell/PowerShell#24490] [General]
Hello.

I have installed PS 7.2.24 on my Windows 7 machine, but I have a problem with _Set-NetConnectionProfile_ command. I wanted to change my network location using this command, but it tells me that it's not recognized. What should I do to make this command available?

**Comment by rhubarb-geek-nz** [comment]
[Set-NetConnectionProfile](https://learn.microsoft.com/en-us/powershell/module/netconnection/set-netconnectionprofile) is not part of PowerShell 7, it is provided by PowerShell 5

```
PS> (Get-Command Set-NetConnectionProfile).Module.ModuleBase
C:\Windows\system32\WindowsPowerShell\v1.0\Modules\NetConnection
```

According to the Microsoft site you would need at least Windows 10 or Windows Server 2012.
